### DISCUSSION TITLE
In-Player Adblock Fixes Lag Issues

### DISCUSSION THREAD

**Discussion by DereC4** [pear-devs/pear-desktop#1183] [General]
Referencing[ a comment ](https://github.com/th-ch/youtube-music/issues/1162#issuecomment-1567498209)on #1162 , it seems toggling adblock to "in-player" fixes many of the lag issues. 

![image](https://github.com/th-ch/youtube-music/assets/53978637/b4beecd2-8113-4038-92ba-713b50e7eb6a)

Am not 100% sure if this does fix or why it does, so if anyone could test it and monitor performance that would be more evidence.

**Comment by yuzhe0690** [comment]
I disabled hw accel and it didnt really lagged anymore

**Reply by DereC4** [reply]
lovely. I did some experiments and have results I will make an issue about in a moment
